app/models/course_year.rb

```diff
@@ -41,18 +43,14 @@ def modules_with_progress(user, modules_to_order = course_modules)
     compute_user_course_module_progress(lessons_with_progresses, modules_in_order)
   end
 
-  def core_induction_programme
-    core_induction_programme_one || core_induction_programme_two
-  end
-
   def title_for(user)
     return title if mentor_title.blank?
 
     user.mentor? ? mentor_title : title
   end
 
   def to_param
-    "year-#{self == core_induction_programme.course_year_one ? '1' : '2'}"
+    "year-#{position}"
```

Comment:
where is position coming from? is it the acts as list that returns it?

Reply:
It is a field in a database, a migration is adding it :)

Reply:
ah yeah, hadn't spotted that column